### PR TITLE
add staff suggestions to the Results page

### PR DESCRIPTION
**Pull request by RCode-Blue**
## Overview of the Pull Request:
Added Staff Suggestions section including infographic to the Results page

## How Has This Been Tested?
There should be a new section at the bottom of the Results page called "Staff Suggestions" that has a description text and an infographic below:

image

## Pull Request Checklist:
- [X] Have you included a link Trello card / Github issue and written sufficient description to help others review your work?
- [X] Is your pull request up-to-date with `main` branch?
- [X] Have you checked That code is well formatted? Did `npx prettier --check .` return no warnings/errors?
- [X] Have you written instructions on how to test that your changes work as expected?
- [X] Have you tested your work thoroughly on your local machine to ensure you are not breaking anything?

### CODE REVIEW

File: components/ResultsPageComponents/StaffSuggestions/StaffSuggestions.jsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+      justify="center"
+    >
+      <Flex direction="column">
+        <Text fontWeight={600} fontSize="33px" lineHeight="37px" py="15px">
+          Staff Suggestions:
+        </Text>
+        <Text fontSize="19px" py="15px">
+          The following graph illustrates the survey participants suggestions on
+          what can council do to support and incentivise staff to choose more
+          sustainable modes of transport to and from work.
```
Comment (hqtan) [inline components/ResultsPageComponents/StaffSuggestions/StaffSuggestions.jsx:24]: please use the following text:
```suggestion         
          Participants were provided with an option to suggest ways that Council
          can support and incentivise staff to choose more sustainable  modes of
          transport to and from work.
```

File: components/ResultsPageComponents/StaffSuggestions/StaffSuggestions.jsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        </Text>
+        <Text fontSize="19px" py="15px">
+          These numbers show the percentage of all people in the survey who
+          suggested this improvement. Some people are in more than one category,
+          and some people didn&#39;t have any suggestions - that&#39;s why the
+          numbers don&#39;t add up to 100%. A stand out in the responses is the
+          suggestions to improving Cycling/Walking infrastructure, increase
+          Public Transit and End-of-Trip facilities at workplace.
```
Comment (hqtan) [inline components/ResultsPageComponents/StaffSuggestions/StaffSuggestions.jsx:32]: please use the following text:
```suggestion          
          A stand out in the responses are suggestions to improve Cycling/Walking
          infrastructure, End-of-Trip facilities at workplace, and Public
          Transport options. The following graph groups participants’ suggestions
          in categories based on the responses received.
```

**Review by hqtan — APPROVED** [review]
Thx for your efforts @RCode-Blue !
🙇🏽 

I'm happy with it!

What do you think @OlaShabalina ?

File: components/ResultsPageComponents/StaffSuggestions/StaffSuggestions.jsx [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+      alignSelf={["center", "start"]}
+      align={["center", "flex-start"]}
+      flex={[1, 2]}
+      direction="column"
+      gap={["10px", "20px"]}
+      px={["5px", "50px"]}
+      py={["25px", "50px"]}
+      justify="center"
+    >
+      <Flex direction="column">
```
Comment (OlaShabalina) [inline components/ResultsPageComponents/StaffSuggestions/StaffSuggestions.jsx:17]: Question about nesting: we have a large Flex container with a lot of attributes, why do we need another Flex container inside it?

Comment (OlaShabalina) [reply components/ResultsPageComponents/StaffSuggestions/StaffSuggestions.jsx:17]: Line 5 then line 17

Comment (hqtan) [reply components/ResultsPageComponents/StaffSuggestions/StaffSuggestions.jsx:17]: that's a really good point @OlaShabalina 👍🏽 
good spot 👍🏽 

I removed the nested Flex container on line 17, and it made no difference to the layout in my local environment.

What do you think @RCode-Blue ?

Comment (RCode-Blue) [reply components/ResultsPageComponents/StaffSuggestions/StaffSuggestions.jsx:17]: Yes, please remove the nested Flex.

Sorry @OlaShabalina you're right, it was unnecessary.

**Review by OlaShabalina — APPROVED** [review]

**Pull request merged**